api/src/main/java/org/pmiops/workbench/api/ClusterController.java

```diff
@@ -55,19 +74,30 @@ public Cluster apply(org.pmiops.workbench.notebooks.model.Cluster firecloudClust
     };
 
   private final NotebooksService notebooksService;
-  private final Provider<User> userProvider;
+  private Provider<User> userProvider;
```

Comment:
Why remove the final on userProvider?

Reply:
See the `setUserProvider()` added below. Unfortunate, but the only way to hack this in for testing.

Reply:
Well, such it is sometimes...

Reply:
Clarification: I shouldn't speak in absolutes. There is probably away to trick Spring into providing a mock value of my choice via `Provider<User>`, but this was beyond my knowledge/patience. We have the same pattern in the WorkspaceControllerTest so both these could be cleaned up if I eventually figure this out.